deploy/kubernetes/metricbeat/metricbeat-daemonset-configmap.yaml

```diff
@@ -85,6 +85,7 @@ data:
         #- /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
     - module: kubernetes
       metricsets:
+        # Not supported on Openshift
```

Comment:
Being this would make the module line have no metricsets enabled, would it be better to place this above `- module: kubernetes`.

Say something along the lines of:

`Currently not support on Openshift, comment out section.`

Reply:
👍